server/server.go

```diff
@@ -158,6 +160,10 @@ func (s *Server) HTTPHandler() http.Handler {
 	staticHandler := http.StripPrefix(proxy.SingleJoiningSlash(s.BaseURL.Path, "/static/"), http.FileServer(http.Dir(s.PublicDir)))
 	handle("/static/", securityHeadersMiddleware(staticHandler))
 
+	handleFunc("/load-test.sw.js", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could you please explain this, I don't quite understand why there is a backend endpoint to serve the file?

Reply:
Because Service Workers can only intercept and control requests from paths below where they are served. It needs to be served at `<base-path>/load-test.sw.js` so it can intercept the API calls to `/api/kubernetes/...`. Serving it from the normal `<base-path>/static/` doesn't work.

Reply:
I see, could you please add a note in here?

Reply:
Good idea, done.